client/src/components/GirderFileManager.vue

```diff
@@ -32,56 +33,54 @@ export default {
   computed: {
     queryValues: {
       get () {
-        if (this.query && this.query.hasOwnProperty('value') && !this.showPartials) {
+        if (this.query && _.has(this.query, 'value') && !this.showPartials) {
```

Comment:
Looks like we are missing the import for lodash?

Reply:
Fixed